source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRSeparator.h

```diff
@@ -0,0 +1,24 @@
+//
+//  ACRSeparator.h
+//  ADCIOSFramework
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "HostConfig.h"
+#import "SharedAdaptiveCard.h"
+
+@interface ACRSeparator:UIView
+
+@property CGFloat dimension_width;
+@property CGFloat dimension_height;
+@property CGFloat lineWidth;
+@property UILayoutConstraintAxis axis;
+@property long    rgb;
```

Comment:
These don't need to be properties, because they're not part of the class's public interface. They should be member variables instead.